plugins/woocommerce/client/admin/client/settings-payments/components/incentive-banner/incentive-banner.tsx

```diff
@@ -0,0 +1,101 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import interpolateComponents from '@automattic/interpolate-components';
+import { Button, Card, CardBody, CardMedia } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { useState } from '@wordpress/element';
+import { Link, Pill } from '@woocommerce/components';
+
+/**
+ * Internal dependencies
+ */
+import './incentive-banner.scss';
+import IncentiveIcon from './incentive-icon.svg';
+import { StatusBadge } from '~/settings-payments/components/status-badge';
+
+export const IncentiveBanner = () => {
+	const [ isSubmitted, setIsSubmitted ] = useState( false );
+	const [ isDismissClicked, setIsDismissClicked ] = useState( false );
+
+	const handleSetup = () => {
+		console.log( 'Handle setup' );
+		setIsSubmitted( true );
+	};
+
+	const handleDismiss = () => {
+		console.log( 'Handle dismiss' );
+		setIsDismissClicked( true );
+	};
+
+	return (
+		<Card className={ 'woocommerce-incentive-banner' } isRounded={ true }>
```

Comment:
Since we aren't expecting its CSS classes from a variable, we could directly use a literal here.

```suggestion
		<Card className="woocommerce-incentive-banner" isRounded={ true }>
```